buildpipeline/Core-Setup-PortableLinux-x64-BT.json

```diff
@@ -0,0 +1,942 @@
+    },
+    "CONNECTION_STRING": {
+      "value": "PassedViaPipeBuild"
+    },
+    "DEB_REPO_PASSWORD": {
+      "value": null,
+      "isSecret": true
+    },
+    "PB_DebianId_ubuntu1404-x64": {
+      "value": null,
+      "isSecret": true
+    },
+    "PB_DebianId_debian8-x64": {
+      "value": null,
+      "isSecret": true
+    },
+    "PB_DebianId_ubuntu1604-x64": {
+      "value": null,
+      "isSecret": true
+    },
+    "PB_DebianId_ubuntu1610-x64": {
+      "value": null,
+      "isSecret": true
+    },
+    "PUBLISH_TO_AZURE_BLOB": {
+      "value": "true",
+      "allowOverride": true
+    },
+    "DOTNET_BUILD_CONTAINER_TAG": {
+      "value": "core-setup-$(PB_DockerOS)-$(Build.BuildId)"
+    },
+    "PB_DockerOS": {
+      "value": "debian8",
+      "allowOverride": true
+    },
+    "PB_CleanAgent": {
+      "value": "true"
+    },
+    "PB_DockerHost_ToolsDirectory": {
+      "value": "$(PB_DockerHost_Sandbox)/Tools"
+    },
+    "PB_DockerImageName": {
+      "value": "$(PB_DockerRepository):$(PB_DockerTag)"
+    },
+    "PB_DockerRepository": {
+      "value": "microsoft/dotnet-buildtools-prereqs"
+    },
+    "PB_DockerContainerName": {
+      "value": "coresetup-$(Build.BuildId)"
+    },
+    "PB_DockerTag": {
+      "value": "ubuntu1404_prereqs_v3"
+    },
+    "OfficialBuildId": {
+      "value": "$(Build.BuildNumber)"
+    },
+    "PB_GitDirectory": {
+      "value": "/root/coresetup"
+    },
+    "SourceVersion": {
+      "value": "HEAD"
+    },
+    "ROOTFS_DIR": {
+      "value": "/crossrootfs/$(PB_TargetArchitecture)"
+    },
+    "PB_CrossBuildArgs": {
+      "value": "",
+      "allowOverride": true
+    },
+    "DockerCommonRunArgs": {
+      "value": "--name $(PB_DockerContainerName) -v \"$(PB_SourcesDirectory):$(PB_GitDirectory)\" -w=\"$(PB_GitDirectory)\" $(PB_DockerImageName)"
+    },
+    "PB_VsoRepoUrl": {
+      "value": "--branch $(PB_Branch) https://$(PB_VsoAccountName):$(PB_VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/DotNet-Core-Setup-Trusted"
+    },
+    "PB_VsoAccountName": {
+      "value": "dn-bot"
+    },
+    "PB_VsoPassword": {
+      "value": null,
+      "isSecret": true
+    },
+    "PB_SourcesDirectory": {
+      "value": "$(Build.SourcesDirectory)/core-setup"
+    },
+    "PB_Branch": {
+      "value": "master"
+    },
+    "PB_DockerHost_Sandbox": {
+      "value": "$(Build.StagingDirectory)/HostSandbox"
+    },
+    "PB_AzureAccountName": {
+      "value": "sourcebuild"
+    },
+    "PB_AzureAccessToken": {
+      "value": null,
+      "isSecret": true
+    },
+    "PB_DistroRid": {
+      "value": "ubuntu.14.04-x64"
+    },
+    "PB_TargetArchitecture": {
+      "value": "x64"
+    },
+    "PB_AdditionalBuildArguments": {
+      "value": ""
+    },
+    "PB_PortableBuild": {
+      "value": "false"
+    },
+    "PB_DebRepoUser": {
+      "value": "dotnet"
+    },
+    "PB_DebRepoServer": {
+      "value": "azure-apt-cat.cloudapp.net"
+    },
+    "PackageTargetsForInstallers": {
```

Comment:
Calling specific set of required targets instead of filtering the targets. Is this okay ?